content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -222,222 +206,240 @@ On Windows, you can use the following settings to configure Services and load ba
+* `V1.Container.ResourceRequirements.limits.cpu` and V1.Container.ResourceRequirements.limits.memory` - Windows doesn't use hard limits for CPU allocations. Instead, a share system is used.
+  The fields based on millicores are scaled into relative shares that are followed by the Windows scheduler. [see: kuberuntime/helpers_windows.go](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/kuberuntime/helpers_windows.go), [see: resource controls in Microsoft docs](https://docs.microsoft.com/en-us/virtualization/windowscontainers/manage-containers/resource-controls)
+* Huge pages are not implemented in the Windows container runtime, and are not available. They require [asserting a user privilege](https://docs.microsoft.com/en-us/windows/desktop/Memory/large-page-support) that's not configurable for containers.
+* `V1.Container.ResourceRequirements.requests.cpu` and V1.Container.ResourceRequirements.requests.memory` - Requests are subtracted from node available resources, so they can be used to avoid overprovisioning a node. However, they cannot be used to guarantee resources in an overprovisioned node. They should be applied to all containers as a best practice if the operator wants to avoid overprovisioning entirely.
+* `V1.Container.SecurityContext.allowPrivilegeEscalation` - not possible on Windows, none of the capabilities are hooked up
+* `V1.Container.SecurityContext.Capabilities` - POSIX capabilities are not implemented on Windows
+* `V1.Container.SecurityContext.privileged` - Windows doesn't support privileged containers
+* `V1.Container.SecurityContext.procMount` - Windows doesn't have a `/proc` filesystem
+* `V1.Container.SecurityContext.readOnlyRootFilesystem` - not possible on Windows, write access is required for registry & system processes to run inside the container
+* `V1.Container.SecurityContext.runAsGroup` - not possible on Windows, no GID support
+* `V1.Container.SecurityContext.runAsNonRoot` - Windows does not have a root user. The closest equivalent is ContainerAdministrator which is an identity that doesn't exist on the node.
+* `V1.Container.SecurityContext.runAsUser` - not possible on Windows, no UID support as int.
+* `V1.Container.SecurityContext.seLinuxOptions` - not possible on Windows, no SELinux
+* `V1.Container.terminationMessagePath` - this has some limitations in that Windows doesn't support mapping single files. The default value is `/dev/termination-log`, which does work because it does not exist on Windows by default.
```

Comment:
since we now have single file mapping, I believe this should be available in containerd deployments 1.21+

Reply:
Can you suggest the replacement text @jsturtevant? I don't think I can work out what this should say instead.

Reply:
@claudiubelu do you know if this `termination-log` works now with single file mapping containerd?

Reply:
Yep, it does. We've also added support for this in Kubernetes v1.17: https://github.com/kubernetes/kubernetes/pull/83057